lib/europeana/blacklight/routes.rb

```diff
@@ -1,24 +1,16 @@
 module Europeana
   module Blacklight
     ##
-    # URL routing for Blacklight
-    module Routes
-      extend ActiveSupport::Concern
-
-      included do |klass|
-        klass.default_route_sets -= [:solr_document]
-        unless klass.default_route_sets.include?(:europeana_document)
-          klass.default_route_sets += [:europeana_document]
-        end
+    # URL routing for Europeana records
+    class Routes
+      def initialize(defaults = {})
+        @defaults = defaults
       end
 
-      def europeana_document(primary_resource)
-        add_routes do |options|
-          args = { only: [:show] }
-          args[:constraints] = options[:constraints] if options[:constraints]
-
-          post 'record/*id/track', args.merge(to: "#{primary_resource}#track", as: 'track_document')
-          get 'record/*id', args.merge(to: "#{primary_resource}#show", as: 'document')
+      def call(mapper, options = {})
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used.